merlin-framework-processor/src/main/java/gov/nasa/jpl/aerie/merlin/processor/metamodel/ActivityTypeRecord.java

```diff
@@ -13,4 +15,12 @@ public record ActivityTypeRecord(
     MapperRecord mapper,
     ExportDefaultsStyle defaultsStyle,
     Optional<EffectModelRecord> effectModel
-  ) implements ExportTypeRecord { }
+  ) implements ExportTypeRecord {
+  public TypeName getOutputTypeName() {
```

Comment:
I like this `getOutputTypeName` method - I think there are a few other places that would benefit from it. (Perhaps it's worth splitting into a refactor commit?)

Reply:
Yes, I agree. Unfortunately, the whole annotation processor is getting quite hairy, and I'd like to take more drastic action against it. I figured for this PR it would be better to avoid making too much of my own mess, and then come back later to improve the whole thing. Does that seem fair to you?

Reply:
Sounds good :+1: